docs/source/using/addons.adoc

```diff
@@ -109,7 +109,8 @@ This can be useful in cases where you know that a command such as `oc` might tak
 
 [NOTE]
 ====
-Trying to use an undefined command will cause an error when the add-on gets parsed.
+- Trying to use an undefined command will cause an error when the add-on gets parsed.
+- Execution failure of command starting with `!` is ignored 
```

Comment:
Two comments
1. The description of `!` is not enough. Adding `!` makes a command execution idempotent i.e. a command can be executed multiple times without changing the final behavior of the add-on. 

2. The note should be with `Comment lines, starting with the '#' character, can be inserted anywhere in the file.`. Line number 42

Reply:
@budhrg @robin-owen I am fine with a separate section too. But may be once we add more syntax in add-on.

Reply:
> The note should be with Comment lines, starting with the '#' character, can be inserted anywhere in the file.. Line number 42

Let's add the `!` note here as it is more visible than the current form. 

> I am fine with a separate section too. But may be once we add more syntax in add-on.

Sure.